spec/rails_admin/support/csv_converter_spec.rb

```diff
@@ -32,10 +32,23 @@
 
     subject { RailsAdmin::CSVConverter.new(objects, schema).to_csv(encoding_to: encoding) }
 
+    context 'when encoding FROM MySQL utf8mb4' do
```

Comment:
Thanks for pointing me in the right direction. I was trying something more complex to solve this problem (something like skip unless connection.adapter_name == 'mysql2' ... which still has an advantage or two).

I've added the attribute to the `it` below because I think it reads better (you can read the 'when ... it ...' without interruption) and Travis is checking my work again and marking some mongoid scenarios green as I write this.

Thanks for your quick response and please do let me know if there's any other way you'd like this pull request improved.